### PR TITLE
WIP: Add flatpak manifest

### PR DESCRIPTION
**Pull request by albfan**
- [x] Added flatpak manifest
- [x] Renamed resources to akiraux
- [ ] Rename license
- [ ] Rename third party (travis, forum)

### CODE REVIEW

File: com.github.akiraux.akira.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      "/share/vala",
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--share=network",
```
Comment (bilelmoussaoui) [inline com.github.akiraux.akira.json]: Why we would need network access here?

File: com.github.akiraux.akira.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      "--filesystem=host",
```
Comment (bilelmoussaoui) [inline com.github.akiraux.akira.json]: The app allows to open just one file at once or multiple files, but we don't open folders or anything like that. We just need to use the NativeFileChooser to use portals and remove this permission

File: data/com.github.akiraux.akira.appdata.xml.in [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright (c) 2017 Alessandro Castellani (https://github.com/Alecaddd/akira) -->
+<!-- Copyright (c) 2017 Alessandro Castellani (https://github.com/akiraux/akira) -->
 <component type="desktop">
```
Comment (bilelmoussaoui) [inline data/com.github.akiraux.akira.appdata.xml.in]:
```suggestion
<component type="desktop-application">
```

File: data/com.github.akiraux.akira.appdata.xml.in [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
-  <id>com.github.alecaddd.akira.desktop</id>
+  <id>com.github.akiraux.akira.desktop</id>
```
Comment (bilelmoussaoui) [inline data/com.github.akiraux.akira.appdata.xml.in]:
```suggestion
  <id>com.github.akiraux.akira</id>
```
and add a launchable tag there.

File: com.github.akiraux.akira.json (hunk 2 above)
Comment (albfan) [reply com.github.akiraux.akira.json]: I'm thinking about config files for akira or similar

Comment (bilelmoussaoui) [reply com.github.akiraux.akira.json]: Config files are stored under `.config` and apps have the rights for that.

File: com.github.akiraux.akira.json (hunk 1 above)
Comment (albfan) [reply com.github.akiraux.akira.json]: related with xmlns or similar, but we can avoid right now

File: com.github.akiraux.akira.json (hunk 2 above)
Comment (albfan) [reply com.github.akiraux.akira.json]: ok

**Comment by Alecaddd** [comment]
I took care of renaming everything with the new RDNN.
If you could fix the merging issues I'd love to merge the flatpak goodness

**Comment by albfan** [comment]
it should be done now.

**Pull request merged**